application/validators/age_validator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative '../../dependencies'
+
+class AgeValidator
+  attr_accessor :errors, :age
+
+  def initialize(age)
+    @errors = []
+    @age = age
+  end
+
+  def valid?
+    @errors << I18n.t(:age_between_23_and_90) unless (Account::VALID_RANGE[:age]).cover?(@age)
```

Comment:
you can make translations more reusable
`I18n.t(:age_between, min: Account::VALID_RANGE[:age][0], max: Account::VALID_RANGE[:age][1])`

Reply:
i did differently - I18n.t(:age_between, number: Account::VALID_RANGE[:age].min && Account::VALID_RANGE[:age].max)

Reply:
changed again - I18n.t(:age_between, min: Account::VALID_RANGE[:age].min, max: Account::VALID_RANGE[:age].max)